labelbox/client.py

```diff
@@ -939,3 +940,306 @@ def get_model_run(self, model_run_id: str) -> ModelRun:
             A ModelRun object.
         """
         return self._get_single(Entity.ModelRun, model_run_id)
+
+    def assign_global_keys_to_data_rows(
+            self,
+            global_key_to_data_row_inputs: List[Dict[str, str]],
+            timeout_seconds=60) -> Dict[str, List[Dict[str, str]]]:
+        """
+        Assigns global keys to data rows.
+        
+        Args:
+            A list of dicts containing data_row_id and global_key.
+        Returns:
+            Dictionary containing 'status', 'results' and 'errors'.
+
+            'Status' contains the outcome of this job. It can be one of
+            'Success', 'Partial Success', or 'Failure'.
+
+            'Results' contains the successful global_key assignments, including
```

Comment:
**nit** I think the word `Results` can be a bit misleading. A failure can also be a result. Is there any reason we do not choose to have it as `successes`?

Reply:
spoke offline on this. keeping as `Results`